frontend/website-redesign/src/pages/Testnet.re

```diff
@@ -0,0 +1,166 @@
+        [
+          width(`percent(50.)),
+          alignItems(`flexStart),
+          justifyContent(`flexStart),
+        ],
+      ),
+      selector(
+        "button",
+        [
+          marginTop(`rem(2.)),
+          important(maxWidth(`percent(50.))),
+          width(`percent(100.)),
+        ],
+      ),
+    ]);
+
+  let disclaimer =
+    merge([
+      Theme.Type.disclaimer,
+      style([
+        display(`listItem),
+        listStylePosition(`inside),
+        marginBottom(`rem(14.)),
+        marginTop(`rem(3.)),
+      ]),
+    ]);
+
+  let gradientSectionExpanded =
+    style([
+      height(`auto),
+      width(`percent(100.)),
+      position(`relative),
+      overflow(`hidden),
+      display(`flex),
+      flexWrap(`wrap),
+      marginLeft(`auto),
+      marginRight(`auto),
+      justifyContent(`center),
+    ]);
+
+  let gradientSection =
+    merge([
+      gradientSectionExpanded,
+      style([
+        height(`rem(53.)),
+        after([
+          contentRule(""),
+          position(`absolute),
+          bottom(`px(-1)),
+          left(`zero),
+          height(`rem(8.)),
+          width(`percent(100.)),
+          pointerEvents(`none),
+          backgroundImage(
+            `linearGradient((
+              `deg(0.),
+              [
+                (`zero, Theme.Colors.white),
+                (`percent(100.), Theme.Colors.white),
+              ],
+            )),
+          ),
+        ]),
+      ]),
+    ]);
+};
+
+[@react.component]
+let make = () => {
+  <Page title="Mina Protocol" footerColor=Theme.Colors.orange>
+    <div className=Styles.page>
+      <div className=Nav.Styles.spacer />
+      <Hero
+        title="Testnet"
+        header="Secure the Network"
+        copy={j|"Push the boundaries of Mina’s testnet to help prepare for mainnet."|j}
+        backgroundImg="/static/img/TestnetBackground.png">
+        <p className=Styles.statusBadge>
+          {React.string("Testnet Status: ")}
+          <StatusBadge service=`Network />
+        </p>
+      </Hero>
+      <Wrapped>
+        <div className=Styles.leaderboardTextContainer>
+          <h2 className=Theme.Type.h2>
+            {React.string("Testnet Leaderboard")}
+          </h2>
+          <p className=Theme.Type.paragraph>
+            {React.string(
+               "Mina rewards community members for contributing to Testnet with Testnet Points, making them stronger applicants for the Genesis Program. ",
+             )}
+          </p>
+          <Button bgColor=Theme.Colors.orange href="/leaderboard">
+            {React.string("See The Full Leaderboard")}
+            <Icon kind=Icon.ArrowRightSmall />
+          </Button>
+        </div>
+        <div className=Styles.gradientSection>
+          <a href="/leaderboard" className=Styles.leaderboardLink>
+            <Leaderboard interactive=false />
+          </a>
+        </div>
+        <p className=Styles.disclaimer>
+          {React.string(
```

Comment:
the spacing between the table and the disclaimer only needs to be 48px 
![image](https://user-images.githubusercontent.com/12700801/94207305-34980800-fe7c-11ea-973d-2dd22f573b96.png)